### PR TITLE
Desktop version pull request

### PR DESCRIPTION
**Pull request by alphym**
desktop and mobile version

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -583,3 +607,290 @@ form textarea:focus {
...
+    color: #344563;
+  }
+
+  #header-desk-nav {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .main-content {
+    background-image: url('./images/desktop_bg.png');
```
Comment (ha-manel) [inline style.css]: - [ ] Kindly fix the path to the background image here, as I can see from your images it's `deskbg.png` and not `desktop_bg.png` 👍

File: index.html [hunk 2]
```diff
@@ -16,7 +16,7 @@
                 <p class="time">9:41</p>
```
Comment (ha-manel) [inline index.html]: - [ ] Kindly remove the bar above the navbar, it's on the template only to show how the website would look on a phone it's not for coding 😅 

image

File: index.html [hunk 3]
```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                    <div class="project-tag">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
+                    </div>
+                    <div class="project-button">
+                        <button>See Project</button>
+                    </div>
```
Comment (ha-manel) [inline index.html:144]: - [ ] According to the template, there should be more space on the sides of the work cards.
Suggestion: you can get the right width of the card by calculating the `%` from Figma then setting `max-width` property 

## Your design:
image

## Figma design:
image

File: index.html [hunk 4]
```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                    <div class="project-description">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <div class="project-tag">
+                        <ul>
+                            <li>html</li>
+                            <li>css</li>
+                            <li>javascript</li>
+                        </ul>
```
Comment (ha-manel) [inline index.html:140]: - [x] Kindly adjust the spacing between these badges to match the template 👍 

## Your design:
image

## Figma design:
image

File: index.html [hunk 5]
```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                    </div>
+                    <div class="project-button">
+                        <button>See Project</button>
```
Comment (ha-manel) [inline index.html:143]: - [x] Kindly adjust the background color of the buttons to match the template. 

## Your design:
image

## Figma design:
image

File: index.html [hunk 6]
```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                                <p>Html</p>
+                            </div>
+                        </li>
+                        <li>
+                            <div class="list-language">
+                                <img src="./images/css.png" alt="">
+                                <p>CSS</p>
+                            </div>
+                        </li>
+                    </div>
```
Comment (ha-manel) [inline index.html:410]: - [x] Kindly add some spacing between these `div`s to match the template 💯 

## Your design:
image

## Figma design:
image

File: index.html [hunk 7]
```diff
@@ -100,169 +109,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                </div>
+                <div class="about-resume">
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+            </div>
             <div class="details">
```
Comment (ha-manel) [inline index.html:367]: - [x] Kindly add some space to the right of this section or adjust its width to match the template 👍 

## Your design:
image

## Figma design:
image

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi Team👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️

## Required Changes:
_Kindly check the comments below_

## Optional:
- [ ] Please consider using best practices when you write commit messages, they should be capitalized sentences.

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 8]
```diff
@@ -100,169 +95,319 @@ <h1>Am Alphy glad to see you!</h1>
...
+                        <li><a href="#"><img src="./images/linkedin.png"></a></li>
+                        <li><a href="#"><img src="./images/hand.png"></a></li>
+                        <li><a href="#"><img src="./images/twitter.png"></a></li>
+                        <li><a href="#"><img src="./images/m.png"></a></li>
+                    </ul>
+                </div>
+                <div class="about-resume">
+                    <button class="btn-resume">Get my resume</button>
+                </div>
+            </div>
```
Comment (MeqdamAlqudah) [inline index.html:366]: - [ Optional ] Please add some space on top of the JS and HTML icons to match the design 

| Yours  |Figms |
| ------------- | ------------- |
| image | image  |

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @alphym , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
image

## Highlights 
:+1: no linter errors 
:+1: Readme file

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**